pkg/routeagent_driver/main.go

```diff
@@ -134,21 +141,39 @@ func init() {
 		"The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
 }
 
-func annotateNode(clusterCidr []string, cfg *restclient.Config) error {
-	k8sClientSet, err := kubernetes.NewForConfig(cfg)
-	if err != nil {
-		klog.Fatalf("Error building clientset: %s", err.Error())
-	}
-
+func annotateNode(clusterCidr []string, k8sClientSet *kubernetes.Clientset) error {
 	nodeName, ok := os.LookupEnv("NODE_NAME")
 	if !ok {
 		return fmt.Errorf("error reading the NODE_NAME from the environment")
 	}
 
-	err = cniapi.AnnotateNodeWithCNIInterfaceIP(nodeName, k8sClientSet, clusterCidr)
+	err := cniapi.AnnotateNodeWithCNIInterfaceIP(nodeName, k8sClientSet, clusterCidr)
 	if err != nil {
 		return errors.Wrap(err, "error annotating node with CNI interface IP")
 	}
 
 	return nil
 }
+
+func getTCPMssValue(k8sClientSet *kubernetes.Clientset) int {
+	usrForceMssVal := 0
```

Comment:
This is used only in line 164 but not in Line 170. We can either remove it all-together or keep it consistent.

Reply:
We should remove it.